lib/mix/tasks/nerves/deps.get.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Mix.Tasks.Nerves.Deps.Get do
+  use Mix.Task
+
+  import Mix.Nerves.IO
+  
+  def run(_argv) do
+    debug_info("Nerves.Deps.Get Start")
+
+    Mix.Tasks.Nerves.Env.run(["--disable"])
```

Comment:
I think that it would be helpful to have a comment here to say what `--disable` does and also why `Nerves.Bootstrap.check_for_update()` is called. Also, why is check_for_update() called last. It seems like you'd want to do that first.

Reply:
The only reason I put check for update last is so that if there is an update available, it would be the last bit of text that is printed on the console instead of it showing up between the output from `deps.get` and `nerves.artifact.get`. I thought it might get lost in there and people would be less likely to update.

I'll add a note about what `--disable is doing`